setup.py

```diff
@@ -25,8 +25,8 @@
         'git+https://github.com/uc-cdis/dictionaryutils.git@2.0.4#egg=dictionaryutils',
         'git+https://github.com/NCI-GDC/cdisutils.git@863ce13772116b51bcf5ce7e556f5df3cb9e6f63#egg=cdisutils',
         'git+https://github.com/NCI-GDC/psqlgraph.git@1.2.2#egg=psqlgraph',
```

Comment:
Can you update this one too, I think its off by a bit

Reply:
good catch